src/app/shared/pipes/table-cell-shortener.pipe.ts

```diff
@@ -7,10 +7,13 @@ import { Subscription } from 'rxjs';
   standalone: true,
 })
 export class TableCellShortenerPipe implements PipeTransform {
-  transform(value: string): string {
+  transform(value: string | boolean): string {
```

Comment:
I'm not sure why this can be a boolean, as far as i know a table cell never has a boolean value.

Reply:
This issue has been fixed, it had to do with the array it looped over also having a boolean value (checked). I changed the interface around a bit so that this wouldnt cause issues anymore